src/randoop/operation/TypeArgumentList.java

```diff
@@ -0,0 +1,66 @@
+package randoop.operation;
+
+import randoop.types.TypeNames;
+
+/**
+ * TypeArgumentList provides static methods for creating and recognizing strings representing
+ * the type arguments of a method or constructor.
+ * <p>
+ * Type arguments are given as comma separated lists of fully qualified class and primitive type
+ * names.
+ * For example:
+ * <ul>
+ * <li><code>int</code>
+ * <li><code>int,double,java.lang.String</code>
+ * <li><code>randoop.operation.Operation</code>
+ * </ul>
+ * 
+ * @see MethodSignatures
+ * @see ConstructorSignatures
+ *
+ */
+public class TypeArgumentList {
```

Comment:
This name suggests that the class represents a list.  I would call it "TypeArgumentListUtils" or "TypeArgumentUtils" or even "TypeArguments", to emphasize that it is a collection of static methods.